### ISSUE TITLE
Questions about reproduce

### ISSUE THREAD

**Issue by Philosober** [tung-nd/stormer#5]
Hello, I am trying to reproduce the pretrain and fine-tune's results. And, I found there is discrepancy between paper and repository's config. Paper said 128 V100 for training, but the config said 1 device and 1 num_nodes. Could you please provide the batch size and learning rate you train on 128 V100?

**Comment by tung-nd** [comment]
Please use the hyperparameters reported in the paper. The configs provided in the repo are default values only.
